### PR TITLE
Fix respecConfig

### PR DESCRIPTION
**Pull request by eladalon1983**
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/multicapture/pull/6.html" title="Last updated on Oct 12, 2022, 7:49 AM UTC (507c6a5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/multicapture/6/94c5b67...507c6a5.html" title="Last updated on Oct 12, 2022, 7:49 AM UTC (507c6a5)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,30 +5,6 @@
     <title>Multicapture</title>
     <script class="remove" src="multicapture.js" type="text/javascript"></script>
     <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
-    <script class="remove">
```
Comment (yoavweiss) [inline index.html]: It might be better to keep the inline config and drop the JS based one

Comment (eladalon1983) [reply index.html]: Done.

File: index.html [hunk 2]
```diff
@@ -3,14 +3,40 @@
...
+        subjectPrefix: "[multicapture]",
+      };
+    </script>
   </head>
   <body>
     <section id="abstract">
       <h2>Abstract</h2>
       <p>TODO</p>
     </section>
+    <section id="sotd"></section>
```
Comment (eladalon1983) [inline index.html:38]: Drive-by, btw.

**Review by yoavweiss — APPROVED** [review]
LGTM

**Review by shangl — APPROVED** [review]
